### ISSUE TITLE
Won't display any playlists or games if syncMode is set to local

### ISSUE THREAD

**Issue by Ludonaut** [winnitron/WinnitronLauncher#19]
If I add _"syncMode": "local"_ to the "launcher" section of winnitron_options.json, the launcher does indeed skip the startup sync but then fails to display any playlists or games.

See attached screenshot and output_log.txt

[output_log.txt](https://github.com/winnitron/WinnitronLauncher/files/731024/output_log.txt)
![winnitron-syncmodelocal-bug](https://cloud.githubusercontent.com/assets/932664/22310901/2fcac262-e351-11e6-9a07-a08a33ef3218.png)

**Comment by aaronklaassen** [comment]
Were there any playlists/games already downloaded? There aren't any games packaged with the launcher, so if it's never synced at all before there won't be anything to load.

**Comment by mrmwiebe** [comment]
I think he means that he just has local games on his machine.  tbh I hadn't tested that properly yet!  I'll look into it when I have a moment.  Does it work if you remove your API key, keep syncMode at default and just have some local games sitting in folders?

**Comment by Ludonaut** [comment]
Yes, games were already downloaded before. In the scenario above I was still subscribed to a playlist on the network that was synced before I changed syncMode to local (just out of curiosity).

> Does it work if you remove your API key, keep syncMode at default and just have some local games sitting in folders?

Nope, but the launcher goes "Oops, Press Q to Quit" and the log sez:
```
[...]
STATE: activating state Oops
 
(Filename: C:/buildslave/unity/build/artifacts/generated/common/runtime/UnityEngineDebugBindings.gen.cpp Line: 42)

STATE: new state is Oops
 
(Filename: C:/buildslave/unity/build/artifacts/generated/common/runtime/UnityEngineDebugBindings.gen.cpp Line: 42)

Error fetching playlists: 403 Forbidden
 
(Filename: C:/buildslave/unity/build/artifacts/generated/common/runtime/UnityEngineDebugBindings.gen.cpp Line: 42)

getting text: error,fetchPlaylistError
 
(Filename: C:/buildslave/unity/build/artifacts/generated/common/runtime/UnityEngineDebugBindings.gen.cpp Line: 42)

Setting up 1 worker threads for Enlighten.
  Thread -> id: 1a9c -> priority: 1 

```

**Comment by mrmwiebe** [comment]
Ooo this is great info, thank you!  Can you attach that full log file to this thread if you still have it?

**Comment by Ludonaut** [comment]
[output_log.txt](https://github.com/winnitron/WinnitronLauncher/files/731077/output_log.txt)

**Comment by mrmwiebe** [comment]
Fixed with commit 80feb714dcd78d54a53c0e7ea4c724e7855b71e9
